google/cloud/storage/blob.py

```diff
@@ -1049,6 +1049,10 @@ def download_as_string(
     ):
         """Download the contents of this blob as a bytes object.
 
+        .. note::
+           The method only supports `python3`, for python2 it returns data
+           as a string.
```

Comment:
This note is incorrect, or at least confusing:  the return value under Python2 *will* be bytes, because the native `str` type is bytes.